backend/SnackOverflow/src/main/java/ro/utcn/sd/vasi/SnackOverflow/controller/rest/UserController.java

```diff
@@ -0,0 +1,48 @@
+package ro.utcn.sd.vasi.SnackOverflow.controller.rest;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.EventListener;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.web.bind.annotation.*;
+import ro.utcn.sd.vasi.SnackOverflow.dto.UserLoggedInDTO;
+import ro.utcn.sd.vasi.SnackOverflow.event.BaseEvent;
+import ro.utcn.sd.vasi.SnackOverflow.services.UserManagementService;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class UserController {
+    private final UserManagementService userManagementService;
+    private final SimpMessagingTemplate messagingTemplate;
+
+    @GetMapping("/me")
+    public UserLoggedInDTO readCurrentUser() {
+        return UserLoggedInDTO.ofEntity(userManagementService.loadCurrentUser());
+    }
+
+    @PostMapping("/me2")
+    public void asdf(@RequestBody UserLoggedInDTO user) {
+        System.out.println(user);
+    }
+
+
+    //shoud this be post or get or put?
+    @PutMapping("/users/{userId}/ban")
```

Comment:
Put. Also it should not have the /ban portion, you should just send the user DTO with the banned flag to true in the body.